docs/agented-vm-guide.md

```diff
@@ -200,14 +201,31 @@ the public cloud VMs. The wrapper install script sets the Environment variable
 install script from antrea repository and triggers installation.
 
 The wrapper install script requires 4 arguments:
-Namespace            - Specifies the Namespace to be used by the `antrea-agent`.
-It should match with the ExternalNode Namespace.
-Antrea version       - Specifies the Antrea version to be used.
-Kubeconfig           - Provides access to the Kubernetes API server.
-Antrea Kubeconfig    - Provides access to the Antrea API server.
-
-For more information on how to generate the kubeconfig files, please refer to
-antrea [ExternalNode](https://github.com/antrea-io/antrea/blob/main/docs/external-node.md#install-antrea-agent-on-vm)
+
+| Argument          | Purpose                                                                                                    |
+|-------------------|------------------------------------------------------------------------------------------------------------|
+| Namespace         | Specifies the Namespace to be used by the `antrea-agent`. It should match with the ExternalNode Namespace. |
+| Antrea version    | Specifies the Antrea version to be used.                                                                   |
+| Kubeconfig        | Provides access to the Kubernetes API server.                                                              |
+| Antrea Kubeconfig | Provides access to the Antrea API server.                                                                  |
+
+### Generate Kubeconfigs
+
+Nephe also provides a helper script `hack/generate-agent-config.sh` to automate
+the generation of kubeconfig and Antrea kubeconfig. It takes two arguments,
+antrea version and Nephe cluster type. Currently only EKS and AKS clusters are
+supported. The script will configure the Nephe cluster to be ready for agented
+VMs and generate the kubeconfigs under your currently directory.
+
+**Note**: The script only works for `vm-ns` Namespace. For other Namespace
```

Comment:
Why this limitation.. Cant you take namespace as input ?

Reply:
Currently we directly uses RBAC yaml from Antrea tagged branch, and they have vm-ns hard coded in that yaml. I was planning in a future PR we can do something like download that file then replace its namespace value in the script.